src/ccid_env.c

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
The CCID driver already include code from PCSC (in `PCSC/` subdiretory).

I propose to have the `ccid_getenv()` function only in PCSC source code and reuse it for CCID.
The name `ccid_getenv()` could be changed to something more generic.